pkgs/development/libraries/boost/1.76.nix

```diff
@@ -0,0 +1,14 @@
+{ callPackage, fetchurl, fetchpatch, ... } @ args:
+
+callPackage ./generic.nix (args // rec {
+  version = "1.76.0";
+
+  src = fetchurl {
+    urls = [
+      "mirror://sourceforge/boost/boost_1_76_0.tar.bz2"
+      "https://dl.bintray.com/boostorg/release/1.76.0/source/boost_1_76_0.tar.bz2"
+    ];
+    # SHA256 from http://www.boost.org/users/history/version_1_76_0.html
+    sha256 = "0hcc661savk32hx65997p0ss1awj6ala4cmz4w7lbi42x6k7nfgh";
+  };
```

Comment:
Is this new boost release required?

If yes, please do it in another commit prior to the mongo changes; if not, please split it in another PR.

Reply:
Yes this is necessary as it is explicitly stated as requirement in downloaded archive. I've separated this update into a separate commit.